src/chat/_chatwidget.py

```diff
@@ -257,6 +259,10 @@ def on_identified(self):
         # Perform any pending autojoins (client may have emitted autoJoin signals before we talked to the IRC server)
         self.autoJoin(self.optionalChannels)
         self.autoJoin(self.crucialChannels)
+        self.autoJoin(self.auto_join_channels)
+
+        if self.client.useNewbiesChannel and self.client.me.player.number_of_games < 51:
```

Comment:
Give a name to the '51' constant. Dunno where best to put it, you can put it statically in this class for now I guess, or in global scope.

Reply:
done